src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+interface PropsCommentList {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<PropsCommentList> = ({ comments }) => {
+  const hasComments = comments.length > 0;
+
+  return (
+    <div className="className">
+      {hasComments
+        ? (
+          comments.map(comment => (
+            <CommentInfo key={comment.id} comment={comment} />
+          )))
+        : (
+          <b data-cy="NoCommentsMessage">No comments yet</b>
+        )}
```

Comment:
Should be like this, no?
```suggestion
          )}
```